core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java

```diff
@@ -17,23 +17,29 @@
     /**
      * The log level of the root category, which is used as the default log level for all categories.
      *
-     * In addition to the standard JDK log level JBoss Logging also adds the following:
+     * JBoss Logging supports Apache style log levels:
      *
-     * {@link org.jboss.logmanager.Level#FATAL}
-     * {@link org.jboss.logmanager.Level#ERROR}
-     * {@link org.jboss.logmanager.Level#WARN}
-     * {@link org.jboss.logmanager.Level#INFO}
-     * {@link org.jboss.logmanager.Level#DEBUG}
-     * {@link org.jboss.logmanager.Level#TRACE}
+     * * {@link org.jboss.logmanager.Level#FATAL}
+     * * {@link org.jboss.logmanager.Level#ERROR}
+     * * {@link org.jboss.logmanager.Level#WARN}
+     * * {@link org.jboss.logmanager.Level#INFO}
+     * * {@link org.jboss.logmanager.Level#DEBUG}
+     * * {@link org.jboss.logmanager.Level#TRACE}
      *
+     * In addition, it also supports the standard JDK log levels.
+     *
+     * @asciidoclet
      */
     @ConfigItem(defaultValue = "INFO")
     public Level level;
 
     /**
      * The default minimum log level
+     * 
+     * @deprecated this functionality was never implemented, it may be deleted or implemented in a future release.
      */
     @ConfigItem(defaultValue = "INFO")
+    @Deprecated
     public Level minLevel;
```

Comment:
If this was never implemented and may be deleted, why not delete it now? I don’t think anyone is using it since it never worked

Reply:
This was discussed on Zulip and decision to make is deprecated waiting to implement it or remove it.
https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/Logging.3A.20what.20is.20min-level.20used.20to.20.3F

Reply:
Ok thanks